draft-irtf-cfrg-vdaf.md

```diff
@@ -902,6 +902,14 @@ this purpose.
 
   * The number of input shares MUST equal `SHARES`.
 
+Sharding is bound to a specific "application context". The application context
+is a string intended to uniquely identify the higher level protocol that uses
+the DAF. This is intended to ensure that aggregation succeeds only if the
+Clients and Aggregators agree on the application context. (Preparation binds
+the application context, too; see {{sec-daf-prepare}}.) Note that, unlike VDAFs
+({{vdaf}}), there is no explicit signal of disagreement; it may only manifest
+as a garbled aggregate result.
```

Comment:
@divergentdave : identify an "instance" of the higher level protocl, not the higher level protocol itself